docs/source/guide/lre-3-options.md

```diff
@@ -0,0 +1,281 @@
+These terms in the monomial basis are also used to define the rows of the sample matrix used for extrapolation.
+```
+
+
+```{code-cell} ipython3
+from mitiq.lre.multivariate_scaling import get_scale_factor_vectors
+
+scale_factors = get_scale_factor_vectors(circuit, degree = 2, fold_multiplier = 2)
+
+scale_factors
+```
+
+
+In the noise scaled circuits created using the above scale factor vectors:
+
+- The term $1$ in the monomial terms basis corresponds to the `degree = 0` term in the polynomial which is equivalent to
+$\lambda_1^0\lambda_2^0\lambda_3^0\lambda_4^0$ term. Due to this term, the first noise-scaled circuit is unchanged.
+
+- due to the $λ_1$ term in the monomial basis, the second noise-scaled circuit only scales the first layer in the circuit.
+
+- due to the $λ_2$ term in the monomial basis, the next noise-scaled circuit only scales the second layer in the circuit
+
+- and so on.
+
+```{code-cell} ipython3
+print("total number of noise scaled circuits created = ", len(scale_factors))
+```
+
+
+
+
+
+
+As the `fold_multiplier` is changed, the number of scaled circuits remains the same but how the layers are scaled
+is altered.
+```{code-cell} ipython3
+
+
+scale_factors_diff_fold_multiplier = get_scale_factor_vectors(circuit, degree = 2, fold_multiplier = 3)
+
+
+print("total number of noise scaled circuits created with different fold_multiplier = ", len(scale_factors_diff_fold_multiplier))
+
+print("Scale factor for some noise scaled circuit with degree = 2 and fold_multiplier = 2 : ", scale_factors[-2] ,sep="\n")
+print("Scale factor for some noise scaled circuit with degree = 2 but fold_multiplier = 3 : ", scale_factors_diff_fold_multiplier[-2] ,sep="\n")
+```
+
+
+Both the number of noise scaled circuits and scale factor vectors are changed when a different value for `degree` is used while keeping everything else the same.
+
+
+```{code-cell} ipython3
+
+
+scale_factors_diff_degree = get_scale_factor_vectors(circuit, degree = 3, fold_multiplier = 2)
+```
+
+
+```{code-cell} ipython3
+
+
+print("total number of noise scaled circuits created = ", len(scale_factors_diff_degree))
+```
+
+
+Thus, even though `degree` and `fold_multiplier` are required to use {func}`.execute_with_lre`, they function as a tunable
+hyperparameter affecting the performance of the technique.
+
+
+### Noise scaling method
+
+
+The default choice for unitary folding in {func}`.execute_with_lre` and {func}`.multivariate_layer_scaling` is
+{func}`.fold_gates_at_random()`.
+
+
+However there are two other choices as well: {func}`.fold_all()` and {func}`.fold_global()` which can be used for the
+`folding_method` parameter in {func}`.execute_with_lre`.
+
+
+```{code-cell} ipython3
+from mitiq.lre.multivariate_scaling import multivariate_layer_scaling
+from mitiq.zne.scaling import fold_all, fold_global
+
+
+# apply local folding
+local_fold_circ = multivariate_layer_scaling(
+ circuit,
+ degree = 2,
+ fold_multiplier = 2,
+ folding_method = fold_all)[5]
```

Comment:
Why is the `[5]` index being used here (and below)?

Reply:
typo, should have been `[1]`